README.md

```diff
@@ -81,6 +81,7 @@ I would recommend to set these options
       enabledReleaseStages: ['staging', 'production'],
       releaseStage: process.env.NODE_ENV
       appVersion: 'YOUR_VERSION',
+      projectRoot: '/'
```

Comment:
I wouldn't add it like this in the recommend options.
Maybe you can add an additional block of options for this setting

Reply:
Alright. I tried to describe how and when to set `projectRoot`.

It might be a good idea to default `projectRoot` to `__dirname`. This might work in most cases (but I am not sure; this needs more experimentation).

Reply:
Hmm yeah that would actually make sense, but I would start with this implementation and keep it in mind for future changes